Adding and subtracting (fixed, I hope)

johnperry-math opened the pull request:
- I've added a submodule for mew_cats.
- I commented out and removed the submodules
    - `public_finance_2018_2019`, since it no longer exists on GitHub.
    - `first-examples`, since it no longer exists in the repository it apparently used to come from
- I ran a git pull recurse submodules because otherwise the absence of a master branch was making `index.py` go nuts.
- I added a string to `index.py` to help me figure out what was going on with one error.
- I reset several submodules because someone had deleted their files in a previous commit, I guess? I don't know, but the files didn't exist after I did everything else.
- I performed some voodoo with `martinthoma-latex-examples`, changing branches and then updating the submodule

My git says this thing is actually up-to-date and all commits are taken care of and I see `mew_cats` so I *think* it's ready to go.

haraldschilly commented:
Hi, I finally had time to go through this. I don't understand why the cocalc examples are commented out. I added them again and also updated a few repositories. In any case, thanks for the contribution. This will be available in the next few days with the next software update.

johnperry-math commented:
> I don't understand why the cocalc examples are commented out.

They didn't work for me. That's the `first-examples` repository, isn't it? I mentioned in the post that it no longer exists in the repository it used to come from. I don't remember the details now, but it simply wouldn't work when I tried it. If it works for you, great. :-)

> I added them again and also updated a few repositories.

Maybe it's the update that I needed to do.

> This will be available in the next few days with the next software update.

Yay, thanks.
